crates/rolldown/tests/esbuild/default/metafile_various_cases/artifacts.snap

```diff
@@ -33,9 +33,34 @@ snapshot_kind: text
 ```text
 [UNRESOLVED_IMPORT] Warning: Could not resolve extern.css in entry.css
 
+```
+## UNRESOLVED_IMPORT
+
+```text
+[UNRESOLVED_IMPORT] Warning: Could not resolve extern.png in entry.css
+
 ```
 # Assets
 
+## assets/file-JEFwJEkD.file
+
+## copy.js
```

Comment:
Since we will fallback to `js` module type if there is no `loader` is specify, `copy.copy` is used `js` loader to process.